<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GfxPaint::DabEditorWidget</class>
 <widget class="QWidget" name="GfxPaint::DabEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>413</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1">
   <item row="6" column="2">
    <widget class="QDoubleSpinBox" name="originYSpinBox">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-512.000000000000000</double>
     </property>
     <property name="maximum">
      <double>512.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="angleSpinBox">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string>deg</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="spaceLabel">
     <property name="text">
      <string>&amp;Space</string>
     </property>
     <property name="buddy">
      <cstring>spaceComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="typeComboBox">
     <item>
      <property name="text">
       <string>Pixel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Distance</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Buffer</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="originLabel">
     <property name="text">
      <string>&amp;Origin</string>
     </property>
     <property name="buddy">
      <cstring>originXSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="opacityLabel">
     <property name="text">
      <string>&amp;Opacity</string>
     </property>
     <property name="buddy">
      <cstring>opacitySpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="composeModeLabel">
     <property name="text">
      <string>&amp;Compose</string>
     </property>
     <property name="buddy">
      <cstring>composeModeComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="metricLabel">
     <property name="text">
      <string>&amp;Metric</string>
     </property>
     <property name="buddy">
      <cstring>metricComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="pixelSnapXComboBox">
     <item>
      <property name="text">
       <string>Off</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Centre</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Edge</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Both</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Auto</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="widthSpinBox">
     <property name="suffix">
      <string>px</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>512.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="pixelSnapLabel">
     <property name="text">
      <string>&amp;Pixel Snap</string>
     </property>
     <property name="buddy">
      <cstring>pixelSnapXComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QDoubleSpinBox" name="heightSpinBox">
     <property name="suffix">
      <string>px</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>512.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string>&amp;Type</string>
     </property>
     <property name="buddy">
      <cstring>typeComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="opacitySpinBox">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.125000000000000</double>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="2">
    <widget class="QComboBox" name="composeModeComboBox"/>
   </item>
   <item row="10" column="1" colspan="2">
    <widget class="QComboBox" name="blendModeComboBox"/>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="spaceComboBox">
     <item>
      <property name="text">
       <string>Object</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Object Aspect-corrected</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>World</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>View</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="blendModeLabel">
     <property name="text">
      <string>&amp;Blend</string>
     </property>
     <property name="buddy">
      <cstring>blendModeComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="hardnessSpinBox">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.125000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="angleLabel">
     <property name="text">
      <string>&amp;Angle</string>
     </property>
     <property name="buddy">
      <cstring>angleSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="fixedRatioCheckBox">
     <property name="text">
      <string>&amp;Fixed Ratio</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="originXSpinBox">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-512.000000000000000</double>
     </property>
     <property name="maximum">
      <double>512.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="metricComboBox">
     <item>
      <property name="text">
       <string>Euclidean</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Manhattan</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Chebyshev</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Minimum</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Octagonal</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="hardnessLabel">
     <property name="text">
      <string>&amp;Hardness</string>
     </property>
     <property name="buddy">
      <cstring>hardnessSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="sizeLabel">
     <property name="text">
      <string>&amp;Size</string>
     </property>
     <property name="buddy">
      <cstring>widthSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="ratioLineEdit"/>
   </item>
   <item row="7" column="2">
    <widget class="QComboBox" name="pixelSnapYComboBox">
     <item>
      <property name="text">
       <string>Off</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Centre</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Edge</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Both</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Auto</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spaceComboBox</tabstop>
  <tabstop>typeComboBox</tabstop>
  <tabstop>metricComboBox</tabstop>
  <tabstop>widthSpinBox</tabstop>
  <tabstop>heightSpinBox</tabstop>
  <tabstop>fixedRatioCheckBox</tabstop>
  <tabstop>ratioLineEdit</tabstop>
  <tabstop>angleSpinBox</tabstop>
  <tabstop>originXSpinBox</tabstop>
  <tabstop>originYSpinBox</tabstop>
  <tabstop>pixelSnapXComboBox</tabstop>
  <tabstop>pixelSnapYComboBox</tabstop>
  <tabstop>hardnessSpinBox</tabstop>
  <tabstop>opacitySpinBox</tabstop>
  <tabstop>blendModeComboBox</tabstop>
  <tabstop>composeModeComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
